smsdk/client.py

```diff
@@ -157,13 +160,17 @@ def get_data_v1(self, ename, util_name, normalize=True, *args, **kwargs):
             # dict params strictly follow {'key':'value'} format
 
             # sub_kwargs = kwargs
-            if util_name in ['get_cycles', 'get_downtime', 'get_parts']:
+            if util_name in ['get_cycles', 'get_downtime', 'get_parts', 'get_factories', 'get_machines', 'get_machine_types']:
```

Comment:
Unit Tests are a separate PR?

Reply:
A useful combination of tools to see how well the code created has test coverage is:
- coverage
- diff-cover

ex.
```
coverage run -m pytest -vvv -s  tests
coverage xml
diff-cover --compare-branch origin/master --html-report diff-report.html coverage.xml
xdg-open diff-report.html
```

The report given goes off the code vs the diff, but still useful.